pkg/minishift/network/dns/localdns.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dns
+
+import (
+	"fmt"
+
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/docker/machine/libmachine/provision"
+	"github.com/minishift/minishift/pkg/minishift/network"
+
+	configCmd "github.com/minishift/minishift/cmd/minishift/cmd/config"
+	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
+
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+)
+
+type serviceCommander interface {
+	Start() (bool, error)
+	Stop() (bool, error)
+	Restart() (bool, error)
+	Status() bool
+	Reset()
```

Comment:
`Reset` is not used by command and IMO we should remove code related to this.

Reply:
Let's see if we need it or not. ATM it is exprimental